### ISSUE TITLE
XML comment Not Showing on Method Hover

### ISSUE THREAD

**Issue by selfmadecode** [selfmadecode/Naija.LGA.Net#1]
The XML documentation comments are not appearing when hovering over methods in the IDE after the package is installed from the NuGet package

Expected Behavior: When a user installs the NuGet package and hovers over methods in the package, the XML documentation should be displayed

**Comment by ObinnaOkpara** [comment]
This looks like an issue with bundling the documentation while packing the package. I'll investigate. Any additional info?

**Comment by selfmadecode** [comment]
Yes you are right, please go ahead, investigate and fix. Thanks for picking this up.
No additional info, you can create a branch for this and create a PR to master

**Comment by ObinnaOkpara** [comment]
I was just going to push a pr for this, but going through your commits, looks like you've fixed it already.

**Comment by selfmadecode** [comment]
no I have not, I have to revert my changes which didn't fix the issue so that you can resolve it

https://github.com/selfmadecode/Naija.LGA.Net/commit/0d6e9c89f869b05258c42166386216b7895d041f
